auslib/test/admin/views/base.py

```diff
@@ -53,10 +53,27 @@ def setUp(self):
         dbo.permissions.user_roles.t.insert().execute(username="bill", role="releng", data_version=1)
         dbo.permissions.user_roles.t.insert().execute(username="bill", role="qa", data_version=1)
         dbo.permissions.user_roles.t.insert().execute(username="bob", role="relman", data_version=1)
+        dbo.permissions.user_roles.t.insert().execute(username="julie", role="releng", data_version=1)
```

Comment:
She's not used directly, but we need RelEng to have more than one user for the Required Signoffs tests to work. I'll add a comment to that effect.